sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -111,6 +111,19 @@ def test_wordcount_it_with_prebuilt_sdk_container_cloud_build(self):
   @pytest.mark.it_validatescontainer
   def test_wordcount_it_with_use_sibling_sdk_workers(self):
     self._run_wordcount_it(wordcount.run, experiment='use_sibling_sdk_workers')
+  
+  @pytest.mark.it_postcommit
```

Comment:
We need to remove pytest.mark.it_postcommit, otherwise regular postcommit suite will pick it up.